content/documentation/zooming-images.md

```diff
@@ -0,0 +1,207 @@
+                "format": "image/jpeg",
+                "service": {
+                  "@context": "http://iiif.io/api/image/2/context.json",
+                  "@id": "https://data.getty.edu/museum/api/iiif/671108",
+                  "profile": "http://iiif.io/api/image/2/profiles/level0.json"
+                },
+                "width": 9073,
+                "height": 7134
+              }
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+Test it by pasting it into a browser, adding `/info.json`, and hitting enter. If correct, you’ll get [a complete JSON document](https://data.getty.edu/museum/api/iiif/671108/info.json) that includes:
+
+```json
+"@context":"http://iiif.io/api/image/2/context.json"
+"@type":"iiif:Image"
+```
+
+Once you’ve identified the URL to the proper `info.json` file, you can jump to the section below on [displaying IIIF image in your project](#displaying-iiif-images-in-your-project).
+
+{{< q-class "box warning" >}}
+
+- When using external resources, IIIF or otherwise, be aware that changes made to those resources by their hosts can effect the way they display in your project. Always link to reliable, permanent sources, and have a regular maintance schedule in place to check for unforeseen changes that may occur after publication.
+
+{{< /q-class >}}
+
+## Create Your Own IIIF Images
+
+You can create your own deep-zooming IIIF images directly within Quire. But there are a couple things you should know from the outset:
+
+1. **Processing IIIF image tiles is a labor intensive process.** It is highly dependent on the processing power of the computer you’re working on. We’ve found that processing a single 130MB image on an average to good machine can take upwards of 20 minutes or more. Three 6–7MB images may take only two or three minutes. Dedicate time to IIIF processing in batches when you’re not running other software on your computer.
+2. **You may need to setup an additional hosting service for IIIF image files, outside your main project files.** Though small in individual file size, the sheer quantity of files (hundreds and sometimes thousands) that are associated with deep-zooming images can cause preview and build issues within your Quire project. If you have more than around 12 IIIF images you’ll want to store them outside your project as detailed below.
+
+### 1. Prepare High-Resolution Source Images
+
+We recommend using uncompressed, full-qualtiy JPEG (jpg) or JPEG 2000 (jp2/jpf/jpx) files for IIIF processing. TIFs can also work, but are typically much larger file sizes and don’t necessarily provide better end results. Quire can also process SVG and PNG files.
+
+Use the table below to size your source images prior to IIIF processing. Note that images larger than the recommended size for a zoom level will work, but will be downsized to the final image size regardless and will cause the image processing to take longer due to the excess file size.
```

Comment:
Should the "note that ..." section be a box warning?

Reply:
I'm going to reword it a bit and call it a tip, but yes, good call!